NEWS

```diff
@@ -73,6 +73,9 @@ issues]].
   Erez Kugler from TSR Gaash.
 - New table for grade 2 Malay braille (Malaysia) thanks to Herbert
   Koh.
+- Improvements to Hebrew 6-dot thanks to BAUM Engineering.
+  - Fixed wrong opcodes and braille pattern for hebrew digits.
```

Comment:
Can you please rephrase this? It should be only about the behavior of the table, "wrong opcodes" is a implementation detail.

Reply:
Also maybe it's better to talk about the "Israeli" table because that's how we called it before in the NEWS file.